example/cars-composite.jv

```diff
@@ -0,0 +1,83 @@
+// Define a new blocktype named CSVExtractor outside of the pipeline
```

Comment:
If the file is in the example folder it will be published as in the Jayvee documentation (https://jvalue.github.io/jayvee/docs/category/jayvee-examples).

Since this file is only relevant for testing purposes, I'd suggest moving it to `apps/pipeline-visualization/test/assets/nested-composite-blocks.jv` (create folders and rename).